<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectRights</class>
 <widget class="QDialog" name="ProjectRights">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="le_IdProject">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_NameProject">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Проект</string>
   </property>
  </widget>
  <widget class="QTableView" name="tV_listUsers">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>256</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolButton" name="tb_select_plus">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>25</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_OK">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Cancel">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tb_select_minus">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>25</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QToolButton" name="tb_Projects">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>25</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <action name="actionAdd">
   <property name="text">
    <string>Добавить</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Удалить</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="text">
    <string>Свойства пользователя</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
